src/controllers/Book.js

```diff
@@ -0,0 +1,90 @@
+const bcrypt = require('bcrypt-nodejs');
+const TokeController = require('../assets/Token');
+const Book = require('../models/Book');
+
+module.exports = {
+  async index(req, res) {
+    const books = await Book.findAll();
+
+    return res.json(books);
+  },
+
+  async store(req, res) {
+    const id_user = req.userId;
+    const {
+      name,
+      author,
+      rating,
+      description,
+      intention,
+      value,
+      image
+    } = req.body;
+
+    try {  
+      const book = await Book.create({
+        name,
+        author,
+        rating,
+        description,
+        intention,
+        value,
+        image,
+        id_user
+      });
+
+      if(!book)
+        return res.status(400).json({ error: 'Unable to register book' });
+
+        return res.json({
+          book
+        });
+
+    } catch (err) {
+      console.log(err);
+      return res.status(500).json({ error: 'Internal Server Error!'});
+    }
+  },
+
+  async update(req, res) {
+    const { id } = req.params;
+
+    try {
+      const updated = await Book.update(req.body, { where: { id } });
```

Comment:
the filter where needs the `user_id`, because we are updating a book that belongs to a user.